data/overlays/actors/z_lmap_mark.rodata.s

```diff
@@ -0,0 +1,61 @@
+.include "macro.inc"
+
+ # assembler directives
+ .set noat      # allow manual use of $at
+ .set noreorder # don't insert nops after branches
+ .set gp=64     # allow use of 64-bit general purpose registers
+
+.section .rodata
+
+.balign 16
+
+glabel D_80830520
+ .word 0x00000000, 0x00000001, 0x00000002, 0x00000003
+glabel D_80830530
+ .word 0x00000002, 0x00000002, 0x00000002, 0x00000003
+glabel D_80830540
+ .word 0x00000003, 0x00000001, 0x00000000, 0x00000000
+glabel D_80830550
+ .word 0x00000002, 0x00000001, 0x00000000, 0x00000000
+glabel D_80830560
+ .word 0x00000000, 0x00000001, 0x00000002, 0x00000004
+glabel D_80830570
+ .word 0x00000000, 0x00000001, 0x00000002, 0x00000002
+glabel D_80830580
+    .asciz "../z_lmap_mark.c"
+    .balign 4
+
+glabel D_80830594
+    .asciz "../z_lmap_mark.c"
+    .balign 4
+
+glabel D_808305A8
+    .asciz "../z_lmap_mark.c"
+    .balign 4
+
+glabel jtbl_808305BC
+.word L80829C28
+.word L80829C28
+.word L80829C28
+.word L80829C28
+.word L80829C28
+.word L80829C28
+.word L80829C28
+.word L80829C28
+glabel jtbl_808305DC
+.word L8082A044
+.word L8082A044
+.word L8082A044
+.word L8082A044
+.word L8082A044
+.word L8082A044
+.word L8082A044
+.word L8082A044
+.word L8082A044
+.word L8082A044
+.word 0x00000000, 0x00000000, 0x00000000
```

Comment:
This is also a file split, so it should at least be replaced with a `.balign 16`, and probably marked as a file split. And the same would apply to any other split not done in those changes.